src/lib/interp.rs

```diff
@@ -152,7 +150,7 @@ impl VM {
                 Instr::LOAD_GLOBAL(ref name) => panic!("NotYetImplemented"),
                 Instr::STORE_GLOBAL(ref name) => panic!("NotYetImplemented"),
                 Instr::NEW_OBJECT(ref class_name) => {
```

Comment:
Do you still need `ref class_name` here?

Reply:
No I don't, I have removed it.

Reply:
I guess that gave an unused warning? If so, have you dealt with any other warnings?

Reply:
The two main warnings I received now are rust complaining that I haven't CamelCased the bytecode instructions, and unused warnings for the `Node` fields in pattern match arms of the parser. These are needed, so I could go through and prefix their names with a `_`.

Reply:
Let's fix them: we should never merge a PR that has warnings unless there's a very, very good reason for them.

Reply:
Okay sure. Would you also suggest refactoring byte-code instructions into CamelCase to satisfy the compiler warnings? In this case I feel it's a suitable break from convention.

Reply:
Yes, you need to fix the bytecode names too, because otherwise you're going to be plagued by warnings for ever. And the problem with warnings is that some of them are important, so if you leave some pointless ones hanging around, they can drown out the important ones.

Reply:
Okay, I understand. I've made those fixes now and there are no more compiler warnings when building the project